### PR TITLE
[prod-stable] Merge stable to prod-stable

### PR DESCRIPTION
**Pull request by lgalis**

### CODE REVIEW

**Review by eclarizio — APPROVED** [review]
Seems all good to me 👍 
 
@lgalis Do we need to wait on any QE approval or testing pipeline to merge this?

**Comment by lgalis** [comment]
@eclarizio - we can wait for QE before merging. The code will  not be accessible in prod until the cloud services config is merged.

**Comment by lgalis** [comment]
@eclarizio - we should merge this - so the cofig PR can be merged tomorrow morning

**Pull request merged**
